ravenframework/SupervisedLearning/SupervisedLearning.py

```diff
@@ -264,9 +266,11 @@ def train(self, tdict, indexMap=None):
         # valueToUse can be either a matrix (for who can handle time-dep data) or a vector (for who can not)
         if self.dynamicFeatures:
           featureValues[:, :, cnt] = (valueToUse[:, :]- self.muAndSigmaFeatures[feat][0])/self.muAndSigmaFeatures[feat][1]
+          # targetValues[:,cnt] = (targetValues[:]- self.muAndSigmaFeatures[self.target[0]][0])/self.muAndSigmaFeatures[self.target[0]][1]
         else:
           featureValues[:,cnt] = ( (valueToUse[:,0] if len(valueToUse.shape) > 1 else valueToUse[:]) - self.muAndSigmaFeatures[feat][0])/self.muAndSigmaFeatures[feat][1]
-
+    if 'normalizeTargets' in self.info.keys() and self.info['normalizeTargets']==True:
+      targetValues = (targetValues - self.muAndSigmaTargets[self.target[0]][0])/self.muAndSigmaTargets[self.target[0]][1]
```

Comment:
I guess this only works on single target, I think you need to expand for multi-target case.